mypy/build.py

```diff
@@ -861,19 +863,68 @@ def is_file(path: str) -> bool:
     return res
 
 
-def find_module(id: str, lib_path_arg: Iterable[str]) -> Optional[str]:
+USER_SITE_PACKAGES = \
+    'from __future__ import print_function; import site; print(site.getusersitepackages());' \
+    'print(*site.getsitepackages(), sep="\\n")'
+VIRTUALENV_SITE_PACKAGES = \
+    'from distutils.sysconfig import get_python_lib; print(get_python_lib())'
+
+
+def call_python(python_executable: str, command: str) -> str:
+    return subprocess.check_output(python_executable.split(' ') + ['-c', command],
+                                   stderr=subprocess.PIPE).decode()
+
+
+@functools.lru_cache(maxsize=None)
+def get_package_dirs(python_executable: str) -> List[str]:
+    """Find package directories for given python
+
+    This defaults to the Python running mypy.
+    """
+    if python_executable == sys.executable:
+        # Use running Python's package dirs
+        try:
+            user_dir = site.getusersitepackages()
+            return site.getsitepackages() + [user_dir]
+        except AttributeError:
+            # fall back on get_python_lib for virtualenvs
+            return [get_python_lib()]
+    else:
```

Comment:
nit: could drop this `else`, since the above is an early return

Reply:
I prefer it there EIBTI.